src/views/admin/applications/AdminHome.tsx

```diff
@@ -0,0 +1,161 @@
+// in the admin portal.
+export interface SidebarDisplay {
+	name: string;
+	status: Status;
+}
+
+export enum Status {
+	accepted,
+	rejected,
+	waitlisted,
+	pending,
+}
+
+interface User {
+	_id: string;
+	guest_auth?: object;
+	application_data: Applicant;
+	role: string;
+	status: string;
+}
+
+function AdminHome() {
+	const { uid, role } = useContext(UserContext);
+	const isLoggedIn = uid !== null;
+	const hasAccess = role === "admin";
+
+	const [userDisplayed, setUserDisplayed] = useState<number>(0);
+	const [sidebarInfo, setSidebarInfo] = useState<SidebarDisplay[]>([
+		{
+			name: "Peter Anteater",
+			status: Status.pending,
+		},
+	]);
+	const [applications, setApplications] = useState<Applicant[]>([
+		{
+			first_name: "Peter",
+			last_name: "Anteater",
+			email: "panteater@uci.edu",
+			gender: "Male",
+			pronouns: ["he/him/his"],
+			ethnicity: "Anteater",
+			is_18_older: true,
+			university: "UC Irvine",
+			education_level: "Alumni",
+			major: "Mascot",
+			is_first_hackathon: false,
+			portfolio_link: "https://github.com/samderanova",
+			linkedin_link: "https://linkedin.com/in/sam-der",
+			stress_relief_question:
+				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, \
+				sed do eiusmod tempor incididunt ut labore et dolore magna \
+				aliqua. Ut enim ad minim veniam, quis nostrud exercitation \
+				ullamco laboris nisi ut aliquip ex ea commodo consequat. \
+				Duis aute irure dolor in reprehenderit in voluptate velit \
+				esse cillum dolore eu fugiat nulla pariatur. Excepteur sint \
+				occaecat cupidatat non proident, sunt in culpa qui officia \
+				deserunt mollit anim id est laborum.",
+			company_specialize_question:
+				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, \
+				sed do eiusmod tempor incididunt ut labore et dolore magna \
+				aliqua. Ut enim ad minim veniam, quis nostrud exercitation \
+				ullamco laboris nisi ut aliquip ex ea commodo consequat. \
+				Duis aute irure dolor in reprehenderit in voluptate velit \
+				esse cillum dolore eu fugiat nulla pariatur. Excepteur sint \
+				occaecat cupidatat non proident, sunt in culpa qui officia \
+				deserunt mollit anim id est laborum.",
+			resume_url: "",
+			submission_time: "now",
+		},
+	]);
+
+	useEffect(() => {
+		const getUsers = async () => {
+			const res = await axios.get<User[]>("/api/admin/users/");
+			setApplications(res.data.map((app) => app["application_data"]));
+			setSidebarInfo(
+				res.data.map((app) => {
+					const name =
+						app["application_data"].first_name +
+						" " +
+						app["application_data"].last_name;
+					let status;
+					switch (app["status"]) {
+						case "ACCEPTED":
+							status = Status.accepted;
+							break;
+						case "REJECTED":
+							status = Status.rejected;
+							break;
+						case "WAITLISTED":
+							status = Status.waitlisted;
+							break;
+						default:
+							status = Status.pending;
+					}
```

Comment:
Thought: I think we could use a string enum instead of a switch, but I'm not certain of all the implementation details right now.